src/WebJobs.Script.WebHost/Diagnostics/MetricsEventManager.cs

```diff
@@ -31,7 +31,7 @@ public class MetricsEventManager : IDisposable
         public MetricsEventManager(ScriptSettingsManager settingsManager, IEventGenerator generator, int functionActivityFlushIntervalSeconds, int metricsFlushIntervalMS = DefaultFlushIntervalMS)
         {
             // we read these in the ctor (not static ctor) since it can change on the fly
-            appName = GetNormalizedString(settingsManager.GetSetting(EnvironmentSettingNames.AzureWebsiteName)) ?? string.Empty;
+            appName = GetNormalizedString(settingsManager.AzureWebsiteDefaultSubdomain);
```

Comment:
Modified this as `GetNormalizedString` will not return a null string and `?? string.Empty` was redundant.